java/performance/actionsframework/src/org/netbeans/actions/simple/SimpleInvoker.java

```diff
@@ -49,7 +49,7 @@ public String getName() {
         return name;
     }
 
-    private Class clazz = null;
+    privateClass<?> clazz = null;
```

Comment:
Huh?! Surely no longer valid code.

Reply:
I always do an "ant clean ; ant" build before I submit a pull-request. FYI. This file was not getting compiled so this error was never seen.

What's happening is this file is only built as part of the qa-functional tests. So it was not seen..